modules/src/tx_msg.rs

```diff
@@ -1,9 +1,13 @@
 use prost_types::Any;
+use std::{
+    vec::Vec,
+    string::String,
+};
 
 use crate::ics24_host::error::ValidationError;
 
 pub trait Msg: Clone {
-    type ValidationError: std::error::Error;
+    type ValidationError: flex_error::StdErr;
```

Comment:
`flex-error` itself does not really define any error type. It looks to me that the error type here should be `crate::ics24_host::error::ValidationError`.

Reply:
Here you need an Error triat, here I included a self-defined Error Trait in `flex-error`, if this is in the case of no-std, there is also an Error trait containing std. What you said here is directly a `crate::ics24_host::error:: ValidationError` to qualify is it possible?